button/generic-data-pipeline
# Adding indexes to daily tables.

This improves query time on main queries from chartio since it indexes on organization_ids which is a common query pattern.

Steps to test:
1. Run db-migrate to local db.
2. Confirm local db tables have the new keys.
3. Copy the output of `SHOW CREATE TABLE [daily_transactions|daily_snapshot]` from prod.
4. Edit the output copied from step 3 to create a new table in experiment db ( e.g. experiment.wt_daily_snapshot_2017_08_10 )
5. INSERT INTO [experiment table created from step 4] SELECT * FROM [daily_transactions|daily_snapshot].
6. Get the query for "Installs by Publisher Partner" from chartio daily report and run an explain on the query.
7. Run an explain on the same query but going against the experiment table instead.
8. Confirm that the query no longer does a full table scan.
